src/schemas/hierarchical-sets.schema.json

```
@@ -0,0 +1,40 @@
+{
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "$id": "https://github.com/hubmapconsortium/vitessce/#hierarchical-sets",
+    "title": "Vitessce hierarchical sets data",
+    "type": "object",
+    "definitions": {
+      "stringArray": {
+        "type": "array",
+        "items": { "type": "string" }
+      },
+      "colorArray": {
+          "type": "array",
+          "items": { "type": "number", "minimum": 0, "maximum": 255 },
+          "minItems": 3,
+          "maxItems": 3
+      }
+    },
+    "additionalProperties": false,
+    "required": ["datasetId", "setsType", "timestamp", "setsTree"],
+    "properties": {
+      "datasetId": { "type": "string" },
+      "setsType": { "type": "string" },
+      "timestamp": { "type": "string" },
```

http://json-schema.org/latest/json-schema-validation.html#rfc.section.7.3.1
```suggestion
      "timestamp": { "type": "string", "format": "date-time" },
```

---

I tried this but the string returned by `(new Date().toLocaleString())` which I am using does not comply this format

---

Now that I think about it more should this field even be included? I think it may be easier to just have a `name` field that includes a timestamp string by default when exporting from Vitessce, but that can be used for a more general name for example if someone wants to import a tree of sets that is from some analysis done in Python rather than coming from Vitessce originally.

---

Update: I decided to remove `timestamp`.